tests/test_idkbse.py

```diff
@@ -0,0 +1,94 @@
+from conf_util import *
+import json
+
+pytestmark = pytest.mark.dev
+
+html_content_types = [
+    "text/html",
+    "application/xhtml",
+    "*/*",
+]
+
+non_html_content_types = [
+    "application/ld+json",
+    "text/turtle",
+    "application/trig",
+    "application/rdf+xml",
+    "application/json",
+    "text/n3",
```

Comment:
Since N3 isn't (yet) a Recommendation, we probably shouldn't "document" supporting it? (I didn't know we did...)

Reply:
Yeah, wasn't sure. Removed!